web/style/cart.scss

```diff
@@ -0,0 +1,137 @@
+@import "_variables.scss";
+@import "common.scss";
+
+.cart-container {
+  // height: 100vh;
+  // @extend .flex-column;
+  background-color: #eee;
+
+  &--head {
+    @extend .flex-column;

```

Comment:
Such uses of extend makes really large CSS selectors: Just check how big is the selector in generated CSS.